File: Python/1. Two Sum.py
class Solution:
    def twoSum(self, a: List[int], t: int) -> List[int]:
        b = []
        
        for i in range(len(a)-1):
            for j in range(i+1,len(a)):
                if(a[i] + a[j] == t):
                    b.append(i)
                    b.append(j)
                    return b

File: Java/1.Two Sum .java
class Solution {
    public int[] twoSum(int[] a, int t) 
    {
        int []b = new int[2];
        int i,j,n;
        n=a.length;
        for(i=0;i<n-1;i++)
            for(j=i+1;j<n;j++)
                if(a[i]+a[j]==t)
                {
                    b[0]=i;
                    b[1]=j;
                    break;
                }
        return b;
    }
}